samples/advanced/live-data.html

```diff
@@ -0,0 +1,117 @@
+<!doctype html>
```

Comment:
I would remove that example from this PR and focus only on fixing the tooltip bug and unit tests.

Reply:
Agreed